book/src/dev/release-criteria.md

```diff
@@ -0,0 +1,113 @@
+# Go/No-Go Release Criteria
+
+## Alpha Release
+
+In the first alpha release, we want Zebra to participate in the network and replicate the Zcash chain state. We also want to validate proof of work and the transaction merkle tree, and serve blocks to peers.
+
+### System Requirements
+
+Our CI tests that Zebra builds, passes its tests, and syncs to tip on the following configurations:
+* Ubuntu Linux ...
+
+Our CI tests that Zebra builds and passes its tests on the following configurations:
+* recent macOS ...
+* recent Windows ...
+
+### Build Requirements
+
+Our CI builds Zebra with:
+* Rust stable
+* recent llvm
+* recent clang
+* recent libclang
+
+### Supported Platforms
+
+While Zebra is still in alpha, we don't guarantee support for any particular platforms.
+
+But you'll probably get the best results with a recent Ubuntu Linux, or the other platforms that our CI runs on.
+
+### Go/No-Go Criteria
+
+- `zebrad` Functionality
+    - [ ] `zebrad` can sync to mainnet tip
+        - [ ] under excellent network conditions (within 2 - 5 hours)
+        - reasonable and sub-optimal network conditions are not yet supported
+    - [ ] `zebrad` can stay within a few blocks of the mainnet tip after the initial sync
+        - [ ] under excellent network conditions
+        - reasonable and sub-optimal network conditions are not yet supported
+    - [ ] `zebrad` can validate proof of work
+    - [ ] `zebrad` can validate the transaction merkle tree
+    - [ ] `zebrad` can serve blocks to peers
+- `zebrad` Performance
+    - [ ] `zebrad` functionality works on platforms that meet its system requirements
+- Testing
+    - [ ] CI Passes
+        - [ ] Unit tests pass reliably
+        - [ ] Property tests pass reliably
+        - [ ] Acceptance tests pass reliably
+- Implementation and Launch
+    - [ ] All release blocker bugs have been fixed
```

Comment:
My intution is that it would be better to enumerate the bugs we know of rather than have a catch all checklist item.

Reply:
I agree, let's discuss either during the zebra gardening call, a separate meeting, or we can try an async triage style process like we did for the tickets needing triage.

Reply:
I think we should have a GitHub tag for release blocker bugs, rather than repeating that information in another document.

Reply:
+1